.gitconfig

```diff
@@ -21,7 +24,9 @@
 	ldm = diff master --name-only
 	p = push origin
 	pl = pull origin
+	pr = remote prune origin
```

Comment:
This assumes that you are on `origin` for branches, which is the default for git and should fit most use cases